src/utils/Utils.tsx

```diff
@@ -19,8 +19,16 @@ import { RequestError } from '../types/RequestError';
 import { EndpointCloud } from '../types/Tenant';
 import { InactivityStatus } from '../types/Transaction';
 import User, { UserRole, UserStatus } from '../types/User';
+import SiteArea from '../types/SiteArea';
+import Site from '../types/Site';
 import Constants from './Constants';
 import Message from './Message';
+import charging_available from '../../assets/icon/charging_station_available.png';
+import charging_charging from '../../assets/icon/charging_station_charging.png';
+import charging_faulted from '../../assets/icon/charging_station_faulted.png';
+import charging_finishing from '../../assets/icon/charging_station_finishing.png';
+import charging_suspended from '../../assets/icon/charging_station_suspended.png';
+import charging_unavailable from '../../assets/icon/charging_station_unavailable.png';
```

Comment:
Rename the files statusMarkerXXXXX with XXXX the status

Reply:
I didn't understand very well. Do you want me to rename the imports ?